packages/global/components/layouts/content/webinar.marko

```diff
@@ -33,7 +33,7 @@ $ const buttonLabels = { continue: continueLabel };
       <theme-content-page-breadcrumbs section=primarySection />
       <marko-web-content-name tag="h1" block-name=blockName obj=content />
       <theme-page-dates|{ blockName }|>
-        <theme-content-published-node block-name=blockName obj=content />
+        Starts: <marko-web-content-start-date block-name=blockName obj=content format="MMMM Do YYYY, h:mm a"/>
```

Comment:
@Shinsina Is there a way to add the timezone?

Reply:
![Screenshot from 2022-11-29 15-45-19](https://user-images.githubusercontent.com/46794001/204654788-a7927fca-6608-4171-a806-a5215e323b07.png)

https://github.com/parameter1/pmmi-media-group-websites/pull/481/commits/69fa7750ba7a00cf61c2182da274341a92b9e6be